hudi-utilities/src/test/java/org/apache/hudi/utilities/TestMultiFormatTimestampBasedKeyGenerator.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.utilities;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.hudi.DataSourceWriteOptions;
+import org.apache.hudi.common.model.HoodieKey;
+import org.apache.hudi.common.util.SchemaTestUtil;
+import org.apache.hudi.common.util.TypedProperties;
+import org.apache.hudi.utilities.exception.HoodieDeltaStreamerException;
+import org.apache.hudi.utilities.keygen.MultiFormatTimestampBasedKeyGenerator;
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Let us use Junit5 apis here?

Reply:
So...I don't think those were in the 0.5.2 branch...I actually did refactor this for the master version I did...I haven't put in that PR yet.

Reply:
Ok, you can find it in master actually. :) I just now checked these changes are to be merged in release-0.5.2 branch. Looks good with Junit4 in that case.